style.css

```diff
@@ -37,115 +41,120 @@ html {
 	padding: 15px;
 }
 
-.welcome-banner {
-	background-color: #EEEEEE;
-	height: 90vh;
-	
+@media screen and (max-width: 600px) {
+	.cv-button {
+	flex-direction: column;
+	font-size: 2rem;
+	}
 }
 
 .navigation-links {
 	color: #ffffff;
 	text-decoration: none;
 	font-family: 'Varela Round', sans-serif;
 	font-size: 3rem;
-    -webkit-text-stroke-width: 0.5px;
-    -webkit-text-stroke-color: #2866c9;
 }
 
-.hi-there-welcome-text {
-	display:flex;
-	align-items: center;
-	justify-content: center;
+@media screen and (max-width: 600px) {
+	.navigation-links {
+	font-size: 2rem;
+	}
+}
+
+/*navigation*/
+
+/*welcome*/
+
+.welcome-banner {
+	background-color: #EEEEEE;
+	height: 90vh;
```

Comment:
90vh creates a lot of white space under your banner on mobile and not at the top, you might want to centre your content or find another way to constrain height
![image](https://user-images.githubusercontent.com/45392468/149968755-73f64a9b-a15c-4e53-b063-9f7ca92609a4.png)